recipes/rosella/build.sh

```diff
@@ -22,4 +22,4 @@ cargo install --path ./ --force --root $PREFIX
 
 # Install flight
 # cd flight/ && pip install . && cd ../
-pip install git+https://github.com/rhysnewell/flight
+#pip install git+https://github.com/rhysnewell/flight
```

Comment:
Yeah, that wouldn't work well and is totally reproducible. Can you just delete the entire "Install flight" section? It should have never been there to begin with.

Reply:
Done, and yeah lesson learned